### PR TITLE
Update totalview package.py with new version and maintainer

### PR DESCRIPTION
**Pull request by dshrader**

### CODE REVIEW

**Comment by tldahlgren** [comment]
@nicholas-sly Did you want to comment on this PR since you were listed as the maintainer?

**Comment by nicholas-sly** [comment]
As one that was involved in the creation of this package definition, I can understand that it's being pushed up from the TCE2 packages.  I think there is room for more development here, but that could just be an issue that is created.  Namely, I think there are more versions that could be added.  That said, I fully grant that this is difficult to set up as the process of obtaining totalview source for the purpose of providing the checksums is not simple.  I'd also be interested in any feedback on how it has been set up (e.g. - using the documentation tarball as the core commonality between all architectures and then architecture-specific installations dependent on the architecture of the system on which it is being installed).

While I was named as the maintainer, I was basing the build definition on David's expertise, so I fully support him being the maintainer going forward.

**Comment by dshrader** [comment]
At the time I did my edits, I didn't know it was possible to put in multiple maintainers. @nicholas-sly, if you would like to still be a maintainer for this, I more than happy to put your name back in.

Thanks,
David

**Comment by tldahlgren** [comment]
@dshrader @nicholas-sly Do you want the maintainers updates before this is merged or is this ready to be merged?

File: var/spack/repos/builtin/packages/totalview/package.py
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/totalview/package.py]: New versions are added to the top of the version directive section:

```suggestion
    version('2021.1.16',
            sha256='4c51c7b6ab6b6afa7635ba2e9fc3b0ef833806f775a0ad0da26b13d6320625dd')
    version('2019.2.18',
            sha256='09e5c554032af945f8cf147dd548421267e50e906cc9686fb5cd0e8e63fcf650')
```

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/totalview/package.py]: I cannot readily confirm the sha256.

**Comment by dshrader** [comment]
Hey @nicholas-sly, would you like to be kept on the maintainers list? If so, I'll update and commit accordingly.

**Comment by nicholas-sly** [comment]
No need.  David is the expert, so he should be the maintainer.

**Review by nicholas-sly — APPROVED** [review]

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**
